rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Volumes in Rancher
+layout: rancher-default-v1.6
+version: v1.6
+lang: en
+---
+
+## Volumes
+---
+
+### Overview
+
+Persistent volumes are a very important part of a stateful application. Rancher makes working with volumes in multi-host environments a simple process.
+
+#### Terminology
+
+A `volume plugin` and `volume driver` are used interchangeably in Docker and Rancher documentation. They refer to the same thing: a plugin that provides local or shared persistent volumes to a Docker container.
+
+A `storage driver` pertains to how containers and images are stored and managed on your Docker hosts. Examples are: aufs, btrfs, zfs, devicemapper. These drivers are beyond the scope of Rancher. More information is available [here](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/).
+
+### Managing Volumes
+
+In this section you’re going to learn how to create persistent volumes that may be shared between containers.
+
+#### Create a Volume
+
+You can create a volume with the `rancher volume create` command.
+
+```
+$ rancher volume create --driver local app-data
+```
+
+This will create a new local volume with name `app-data`. The name must start with an alphanumeric character, followed by `a-z0-9`, `_` (underscore), `.` (period) or `-` (hyphen). 
+
+The `--driver` flag specifies the volume driver to use. Docker ships with a `local` volume driver. Volumes using this driver persist their data to the host filesystem and are accessible to any container on the same host.
+
+#### List Volumes
+
+You can list volumes in an environment.
+
+```
+$ rancher volume ls
+```
+
+If you created the `app-data` volume from the previous section, you may wonder why it wasn't listed. You can include `inactive` volumes with the `--all` or `-a` flag.
+
+```
+$ rancher volume ls --all
+```
+
+#### Delete a Volume
+
+You can delete a volume with the `rancher volume rm` command.
+
+```
+$ rancher volume rm app-data
+```
+
+#### Volume States
+
+Volumes have five distinct states: `inactive`, `activating`, `active`, `detached`, and `removed`.
```

Comment:
Nice, thanks for filling me in on that. I double checked the state for a newly created volume is inactive, so we should be good on that note.